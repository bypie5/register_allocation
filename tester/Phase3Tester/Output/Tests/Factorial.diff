--- SelfTestCases/Factorial.out	2020-03-05 22:31:29.000000000 -0800
+++ ./Output/Tests/Factorial.out	2020-03-08 22:08:58.000000000 -0700
@@ -1 +1,13 @@
-3628800
+--- Error (ProgramFault) ---
+expecting argument 2 to be an integer, found data address<Heap.0>+0
+  Main() line 17
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 43
+  Fac.ComputeFac() line 45
