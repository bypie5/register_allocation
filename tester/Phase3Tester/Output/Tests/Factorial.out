--- Error (ProgramFault) ---
expecting argument 2 to be an integer, found data address<Heap.0>+0
  Main() line 17
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 43
  Fac.ComputeFac() line 45
